internal/provider/resource_packer_channel.go

```diff
@@ -95,11 +104,6 @@ func resourcePackerChannel() *schema.Resource {
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
-			"project_id": {
-				Description: "The ID of the project this HCP Packer registry is located in.",
```

Comment:
The field descriptions within this resource appear to be behind a commit https://github.com/hashicorp/terraform-provider-hcp/blob/0d79f9c8040747a24a24df75330f3ad47c9f2f57/internal/provider/resource_packer_channel.go#L101

Is your local branch up to date with what is on main?

Reply:
This PR is going into a feature branch, which I've just rebased: https://github.com/hashicorp/terraform-provider-hcp/pull/454. Hopefully that addresses the issue!

Reply:
Ah, and just pushed the rebase of this branch. Do the latest changes capture all the fixes you were expecting? I referred to this change:
https://github.com/hashicorp/terraform-provider-hcp/commit/c1730879d133ea105c34314806cab01000271a69

Reply:
Haha I might have caused more issues with my rebase 😅 Standby.

Reply:
Hey @nywilken! Wanted to check in to see if you were okay with the changes that were made to bring this PR back up to `main`.